### ISSUE TITLE
Search does not match phone or email, overmatches with space

### ISSUE THREAD

**Issue by nolash** [teamsempo/SempoBlockchain#205]
In `app/server/api/search.api:SearchAPI` we find this description:

```
This endpoint searches transfer accounts and credit transfers. It will check first name/last name/phone number/email address
```

However when I search for a user whom I get returned by search by lastname using its email or phone no results are returned.

Furthermore if I search for first and last name separated by space all matches from both words are returned. When I tried to enclose the name with single apostrophes the instance crashed (which might have been a coincidence).

**Comment by nolash** [comment]
Furthermore, setting up a fresh instance with test data, I cannot get any results at all, even without a search string. A dump of the db with fake data can be found here:

https://swarm-gateways.net/bzz:/ec5ee335d1ccc701a85bb421b0d5e1367d09cd6846f3f5319c7d74f7b40d8e7b/

Any pointers to what additional data that must be present to make searches possible would be much appeciated

**Comment by enjeyw** [comment]
It could possibly be the `org` parameter? For most HTTP requests this needs to be set to the organisation ID that you're querying from:
https://github.com/teamsempo/SempoBlockchain/blob/fba440a1bc9b70423927730292ef19a7ab6886f8/app/server/utils/auth.py#L176
Note this isn't documented currently.

> Furthermore if I search for first and last name separated by space all matches from both words are returned. 
From memory, this was designed behaviour... Checking in with @michielderoos 


> When I tried to enclose the name with single apostrophes the instance crashed (which might have been a coincidence).
I think that's a coincidence - I haven't seen this myself (just tested it)

Overall, tests on this endpoint would definitely be a good todo

**Comment by tristanhcole** [comment]
Closed by #615
